dipy/workflows/reconst.py

```diff
@@ -746,10 +772,12 @@ def run(
         roi_radii=10,
         fa_thr=0.7,
         frf=None,
-        extract_pam_values=False,
-        sh_order=8,
-        odf_to_sh_order=8,
```

Comment:
Was `odf_to_sh_order` not being used at all?

Reply:
No, it was not used, that's why I clean it up and clarify